### PR TITLE
doc-improvement: doc: nrf: Small improvements and fixes for sysbuild doc

### PR DESCRIPTION
**Pull request by MarekPieta**
Change introduces small improvements and fixes for sysbuild doc.

### CODE REVIEW

File: doc/nrf/config_and_build/configuring_app/zephyr_samples_sysbuild.rst [hunk 1]
```diff
@@ -13,7 +13,7 @@ For applications and samples in the |NCS| repository, this is handled automatica
 +---------------------------------------------------------+-------------------------+-----------+----------+-----------------------------------------------------------------------------------------------------------------------+
 | :kconfig:option:`SB_CONFIG_NETCORE_HCI_IPC`             | ``hci_ipc``             | ✓         | ✕        |                                                                                                                       |
 +---------------------------------------------------------+-------------------------+-----------+----------+-----------------------------------------------------------------------------------------------------------------------+
-| :kconfig:option:`SB_CONFIG_NETCORE_RPC_HOST`            | ``rpc_host``            | ✓         | ✕        | Requires that application be setup for this mode                                                                      |
+| :kconfig:option:`SB_CONFIG_NETCORE_RPC_HOST`            | ``rpc_host``            | ✓         | ✕        | Requires that application is setup for this mode                                                                      |
```
Comment (greg-fer) [inline doc/nrf/config_and_build/configuring_app/zephyr_samples_sysbuild.rst:16]: This is gramatically and linguistically correct :) https://en.wikipedia.org/wiki/English_subjunctive

**Review by greg-fer — COMMENTED** [review]
You might want to poke @FrancescoSer about including these edits in #15760 , because he edits these same files there extensively.

File: doc/nrf/config_and_build/configuring_app/zephyr_samples_sysbuild.rst (hunk 1 above)
Comment (MarekPieta) [reply doc/nrf/config_and_build/configuring_app/zephyr_samples_sysbuild.rst:16]: Ok, removing :)

File: doc/nrf/config_and_build/configuring_app/sysbuild_images.rst [hunk 2]
```diff
@@ -187,7 +187,7 @@ Then, add this folder to the Zephyr module file:
 .. code-block:: yaml
 
     build:
-      sysbuild-cmake: sysbuild  # Only needed is a sysbuild CMakeLists.txt file is being added
+      sysbuild-cmake: sysbuild  # Only needed if a sysbuild CMakeLists.txt file is being added
```
Comment (MarekPieta) [inline doc/nrf/config_and_build/configuring_app/sysbuild_images.rst:190]: Hi @FrancescoSer, Can we fix this typo?

File: doc/nrf/releases_and_maturity/migration/migration_sysbuild.rst [hunk 3]
```diff
@@ -658,7 +658,7 @@ As sysbuild deals with configuration of features for some features and propagati
 Incompatibities
 ===============
 
-In the sysbuild release included in the |NCS| 2.7, the following features of the multi-image builds using child an parent images are not supported:
+In the sysbuild release included in the |NCS| 2.7, the following features of the multi-image builds using child and parent images are not supported:
```
Comment (MarekPieta) [inline doc/nrf/releases_and_maturity/migration/migration_sysbuild.rst:661]: Hi @FrancescoSer, this seems like a typo. Can we integrate it to your PR: https://github.com/nrfconnect/sdk-nrf/pull/15760?

File: doc/nrf/releases_and_maturity/migration/migration_sysbuild.rst [hunk 4]
```diff
@@ -451,9 +451,9 @@ The expected output files are the following:
...
 | ``ipc_radio/zephyr/zephyr.bin``                     | Network core IPC radio binary file                                                                    |
 +-----------------------------------------------------+-------------------------------------------------------------------------------------------------------+
-| ``b0n/zephyr/zephyr.bin``                           | Network core secure bootloader hex file                                                               |
+| ``b0n/zephyr/zephyr.bin``                           | Network core secure bootloader binary file                                                            |
 +-----------------------------------------------------+-------------------------------------------------------------------------------------------------------+
-| ``b0n/zephyr/zephyr.hex``                           | Network core secure bootloader binary file                                                            |
+| ``b0n/zephyr/zephyr.hex``                           | Network core secure bootloader hex file                                                               |
```
Comment (MarekPieta) [inline doc/nrf/releases_and_maturity/migration/migration_sysbuild.rst:456]: Hi @FrancescoSer, this seems like a typo. Can we integrate it to your PR: https://github.com/nrfconnect/sdk-nrf/pull/15760?

**Comment by MarekPieta** [comment]
@FrancescoSer cherry-picked my fixes as part of https://github.com/nrfconnect/sdk-nrf/pull/15760. Closing this one.

**Pull request closed without merging**